CONTRIBUTING.md

```diff
@@ -231,6 +232,37 @@ To enabled Hot Reloading:
 
 Read more about [Hot Reloading](./docs/local-development.md#hot-reloading)
 
+### Logging
+
+Logging information can be very useful when developing, and there are a few logging options available to you.
+
+To enable logging:
+
+* [Create a local config file](#create-a-local-config-file) if you don't already have one
+* Edit your local config, changing the value of the logger type you want to see to `true`
+
+```json
+  "logging": {
+    "client": false,
+    "firefoxProxy": false,
+    "actions": true
+  }
+```
+
+Let's cover the logging types.
+
+#### client
+
+This option is currently unused.
+
+#### firefoxProxy
+
+This logger outputs a verbose output of all the Firefox protocol packets to your shell.
+
+#### actions
+
+This logger outputs the Redux actions fired to the browser console.
```

Comment:
I still think a list as I had in my previous review would be even better. This are items that you want to scan, not section headers.